### PR TITLE
No driver

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets |
| License         | MIT

When configuring a resource which is not managed by Doctrine

image

### CODE REVIEW

File: src/Bundle/DependencyInjection/Driver/DriverProvider.php [hunk 1]
```diff
@@ -45,6 +46,8 @@ public static function get(MetadataInterface $metadata): DriverInterface
                 return self::$drivers[$type] = new DoctrinePHPCRDriver();
         }
 
+        Assert::notFalse($type, sprintf('No driver was configured on the resource "%s".', $metadata->getAlias()));
```
Comment (lchrusciel) [inline src/Bundle/DependencyInjection/Driver/DriverProvider.php]: Why do we check it after switch?

File: src/Bundle/test/config/sylius/resources.yaml [hunk 2]
```diff
@@ -3,6 +3,11 @@ sylius_resource:
         locale_provider: test.translation_locale_provider
 
     resources:
+        app.board_game:
+            driver: false
```
Comment (lchrusciel) [inline src/Bundle/test/config/sylius/resources.yaml:7]: Why don't we use `null` here?

Comment (loic425) [reply src/Bundle/test/config/sylius/resources.yaml:7]: null is for default value and this will be configured with doctrine/orm value instead.

Comment (lchrusciel) [reply src/Bundle/test/config/sylius/resources.yaml:7]: But if we have a default value if null will be provided, this false needs to be defined explicitly. And then, we trigger an error. I don't see the use case for this functionality. Can you write some examples of usage of it?

Comment (loic425) [reply src/Bundle/test/config/sylius/resources.yaml:7]: It does not trigger an error
https://github.com/Sylius/SyliusResourceBundle/blob/1fcf7dcac28f26217736cc6febd28c550859d3d3/src/Bundle/DependencyInjection/SyliusResourceExtension.php#L145

**Review by lchrusciel — APPROVED** [review]

**Pull request merged**